package-releases/base/cronjob.yaml

```diff
@@ -16,13 +13,9 @@ spec:
   jobTemplate:
     spec:
       template:
-        metadata:
-          labels:
-            app: thoth
-            component: package-releases
         spec:
           containers:
-            - image: "package-releases-job:latest"
+            - image: "package-releases-job"
```

Comment:
You miss Kafka specific env variables, have a look at package update cronjob :) also how to run producer.py from app.sh :)

Reply:
Thanks, https://github.com/thoth-station/thoth-application/pull/425/commits/33969ce76b8a00688a40479757e497120438cdfd, I think we don't provide `SUBCOMMAND` for value `producer`, as in this case we just need to run `app.sh`.

Reply:
only if there is a defaul value in `app.sh` in package release job, otherwise we have to :)

Reply:
thanks, https://github.com/thoth-station/thoth-application/pull/425/commits/45fbfc8b839c0d24951f26caffddd77854af189a